ogre/src/OgreRTShaderSystem.cc

```diff
@@ -485,9 +485,9 @@ void OgreRTShaderSystem::GenerateShaders(OgreSubMesh *subMesh)
 bool OgreRTShaderSystem::Paths(std::string &coreLibsPath,
     std::string &cachePath)
 {
-  const char *env = std::getenv("IGN_RENDERING_RESOURCE_PATH");
+  const char *env = std::getenv("GZ_RENDERING_RESOURCE_PATH");
```

Comment:
Here too, we have to tick-tock the env var wherever it's used

Reply:
https://github.com/gazebosim/gz-rendering/pull/629/commits/c69cbc52219181c67458a53e95f45161a4e6398b